website/src/pages/v1/serve/deployment/docker.mdx

```diff
@@ -136,50 +136,81 @@ FROM ghcr.io/ardatan/mesh-serve
 RUN npm i @graphql-mesh/plugin-rate-limit
 ```
 
-### Develop additional resolvers
+### Develop plugin
 
-However, you may have additional resolvers and have a more sophisticated setup with some
-dependencies and source code, copying over your project's files is the way to go.
+However, you may be developing a plugin and have a setup with some dependencies and source code,
+copying over your project's files is the way to go.
 
-Say you have the following files:
+In the following example, we're developing a `useTiming` plugin that will add a human readable
+execution duration to the GraphQL result `extensions` property.
 
 ```json filename="package.json"
 {
-  "name": "my-time",
+  "name": "my-timing",
   "dependencies": {
     "moment": "^2"
   },
   "devDependencies": {
-    "@graphql-mesh/serve-cli": "latest"
+    "@graphql-mesh/serve-cli": "latest",
+    "@graphql-mesh/serve-runtime": "latest"
   }
 }
 ```
 
-```js filename="my-time.ts"
+```ts filename="my-timing.ts"
 import moment from 'moment'
-
-export const additionalResolvers = {
-  Query: {
-    formattedToday() {
-      return moment().format('DD.MM.YYYY')
+import type { MeshServePlugin } from '@graphql-mesh/serve-runtime'
```

Comment:
We can export this from CLI package btw.

Reply:
But a Serve Plugin is for serve runtime, not the CLI. The current export path makes it obvious.